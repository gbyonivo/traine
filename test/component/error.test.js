import React from 'react';
import renderer from 'react-test-renderer';
import Error from '../../src/components/error';

describe('Error', () => {
  it('should render correctly', () => {
    const actual = renderer.create(<Error refetch={() => {}} error={'qwqeqfdj dsfljsllsf dlsfjljsf'}/>);
    expect(actual).toMatchSnapshot();
  });
});