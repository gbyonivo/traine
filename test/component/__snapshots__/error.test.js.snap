// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render correctly 1`] = `
<div
  className="error"
>
  qwqeqfdj dsfljsllsf dlsfjljsf
  <br />
  <button
    onClick={[Function]}
  >
    refresh
  </button>
</div>
`;
